.env.example

```diff
@@ -0,0 +1,10 @@
+COFFEE_CONTAINER_PATH=/random_coffee/app
+COFFEE_CONTAINER_NAME=random_coffee
+COFFEE_CONTAINER_LOGS=/random_coffee/app/logs
+COFFEE_CONTAINER_PORT=8080
+COFFEE_DB_HOST=postgres
+COFFEE_DB_PORT_NUMBER=5432
+COFFEE_DB_NAME=coffee_postgres_db
+COFFEE_DB_USER=postgres
+COFFEE_DB_PASSWORD=postgres
+COFFEE_DB_CONTAINER_NAME=coffee_postgres
```

Comment:
If you will leave env example, please make sure, that it does not contain any sensitive data such as username and passwords, replace it with dummy values